### PR TITLE
Add Recurring Scan Feature

### PR DESCRIPTION
**Pull request by ryan-a-baker**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?

Just main

It's best to PR to the most recent relevant branch and leave all the cherry-picking to the
writer, except where earlier branches require factual changes to the content.
For more information about the branches, see https://github.com/pivotal/docs-tap#branches

### CODE REVIEW

**Comment by ryan-a-baker** [comment]
This PR needs https://github.com/pivotal/docs-tap/pull/3251 merged to included the external DB updates before this is merged.

**Comment by kcoriordan** [comment]
@ryan-a-baker can you also check the TOC location where I apped recurring-scanning file. thanks
https://docs-staging.vmware.com/en/draft/VMware-Tanzu-Application-Platform/1.8/tap/scst-scan-recurring-scanning.html

File: scst-scan/recurring-scanning.hbs.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+* Use when either Scan 1.0 or Scan 2.0 is defined as your supply chain scanning component
+
+## <a id="recurring-scanning-setup"></a>Recurring Scanning Setup
+
+In order to enable recurring scanning, you must create a recurringimagevulnerabilityscan.  This will define the following:
+
+* The interval in which to scan container images in crontab format
+* How far back (in days) of images created via the supply chain to scan
+* How far back (in days) of images that have started in your TAP Clusters to scan
+* The steps from [IVS template]() to use to scan your images, which defines what scanner to use
```
[the remark this answers is not in the document]
Comment (ryan-a-baker) [reply scst-scan/recurring-scanning.hbs.md:27]: Added in an upcoming PR.  Good catch.

File: scst-scan/recurring-scanning.hbs.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+### <a id="preqrequisites"></a>Prerequisites
+
+Before you define your recurringimagevulnerabilityscan template, you must have the following
+
+A repository created on an OCI compliant container registry that will be used to push scan results to
+A service account with the ability to push an oci artifact to the results repository
+Credentials for any registry the scanner will need to pull images from to scan
+
+The prerequisites for recurring scan are the same as Scan 2.0.  For service accounts and credentials, you can do this manually following the Scan 2.0 directions, but the recommended approach is to use the  namespace provisioner to create a namespace.  This will create all of the required resources needed for recurring scanning to work.  For the duration of this guide, we will assume namespace provisioner is used.
```
[the remark this answers is not in the document]
Comment (ryan-a-baker) [reply scst-scan/recurring-scanning.hbs.md:40]: I totally re-wrote this intro in the next PR.  Recurring scanning uses Scan 2.0, which is now installed by default.  This section has just been updated to say it's installed with the full and build profile.

File: scst-scan/recurring-scanning.hbs.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+
+### <a id="example-template"></a>Example recurringimagevulnerabilityscan Template
+
+Below is a sample template with an explanation of all the input variables that can be provided for the recurringimagevulnerabilityscan CR.  The samples for Grype and Trivy below are intended to used in a namespace that is created by the namespace provisioner and a simple environment.  As such, the Grype and Trivy examples are a subset of this template.  Additional configurations from this template can be added to the Grype and Trivy samples for more advanced configurations.
```
[the remark this answers is not in the document]
Comment (ryan-a-baker) [reply scst-scan/recurring-scanning.hbs.md:44]: I think it meant not a bunch of container registries.  I'm going to remove that phrase and have two different paths in the document, one for supply chain images and one for all running images.

File: scst-scan/recurring-scanning.hbs.md [hunk 4]
```diff
@@ -0,0 +1,187 @@
+# Recurring Scanning
+
+In a world where new vulnerabilities are reported daily, it is important that workloads are scanned with updated vulnerability databases frequently.  Tanzu Application Platform provides the capability to use the scanner of your choice to scan images produce by your software supply chain, as well as any container image running in your Tanzu Application Platform clusters.
+
+## <a id="overview"></a>Overview
+
+Scan 2.0 provides the ability to schedule container image scans with the following capabilities:
```
[the remark this answers is not in the document]
Comment (ryan-a-baker) [reply scst-scan/recurring-scanning.hbs.md:7]: Ew... I copied and pasted this from word and it messed up the formatting.  Fixing it in my next PR.

**Comment by ryan-a-baker** [comment]
> @ryan-a-baker can you also check the TOC location where I apped recurring-scanning file. thanks https://docs-staging.vmware.com/en/draft/VMware-Tanzu-Application-Platform/1.8/tap/scst-scan-recurring-scanning.html

Looks great.  Thank you!

**Comment by ryan-a-baker** [comment]
All comments addressed in https://github.com/pivotal/docs-tap/pull/3309

**Pull request merged**